src/run/run_step.go

```diff
@@ -25,6 +25,30 @@ import (
 
 var log = logging.MustGetLogger("run")
 
+type ParallelOutput uint
```

Comment:
Might be easier to make this guy a string? Then you wouldn't have to worry about unmarshalling it (combined with the `choice` options above)

Reply:
Agreed, with the `choice` options this gets simplified.